src/integrationTest/resources/application.properties

```diff
@@ -28,3 +28,5 @@ user.timezone=GMT
 
 # no communication with Service Bus
 spring.profiles.active=nosb
+
+reports.recipients=integration@test
```

Comment:
may be use a proper email format ? something like` integration@fake.com`

Reply:
don't want to use domain fully. it is valid to assign partial domain and java mail sender allows it. it's like having you local machine setup with `username@localhost`